src/main/java/com/epam/reportportal/jobs/repository/ProjectRepository.java

```diff
@@ -0,0 +1,42 @@
+package com.epam.reportportal.jobs.repository;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.jdbc.core.namedparam.NamedParameterJdbcTemplate;
+import org.springframework.stereotype.Repository;
+
+import java.util.Collections;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+import static com.epam.reportportal.jobs.repository.RepositoryConstants.*;
+
+/**
+ * @author <a href="mailto:ivan_budayeu@epam.com">Ivan Budayeu</a>
+ */
+@Repository
+public class ProjectRepository {
```

Comment:
no need to use 3 level arch here. classes are pretty simple, so we can use everything in job class